index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+    <title>Code Quiz | Home</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav-container">
+        <div class="nav-item">
+          <a href="./highScores.html">High Scores</a>
+        </div>
+        <div class="nav-item">Time: <span id="countdown">0</span></div>
+      </nav>
+    </header>
+
+    <main class="main" id="main-container">
+      <!-- step1 - this container must disappear / js-->
+      <div class="container start-container" id="start-container"> 
+        <h1 class="title">Geography Quiz</h1>
+        <p>
+          Introduction to geography unit test: quiz. Studying geography gives
+          one the chance to understand the world we live in and the different
+          features within it. Do you know where the tallest mountain in the
+          world can be found, time zones in different parts of the world or how
+          to actually read a map? This quiz will provide you with some insight
+          do give it a try and keep a lookout for more like it!
+        </p>
+        <div>
+          <!--added id to the button to link with js-->
+          <button class="btn" id="start-quiz">Start Quiz</button>
+        </div>
+      </div>
+      <div class="container answer-alert answer-alert-wrong">
```

Comment:
This static HTML needs to be removed as it will be dynamically generated by your JavaScript code.